src/public_html/desktop_settings.html

```diff
@@ -126,6 +126,16 @@ <h5 class="theme-text">Choose your theme highlight color</h5>
                 <label for="last_name">Thumbs Down Current Song</label>
               </div>
             </div>
+            <div class="row">
```

Comment:
I didn't think that this would work without adding extra inject JS to the settings tab, turns out I wrote some funky awesomeness that handles it :+1:  to me :smile: